frontends/web/src/app.tsx

```diff
@@ -83,8 +83,13 @@ export const App = () => {
       return;
     }
     // if no devices are registered on specified views route to /
-    if (Object.keys(devices).length === 0 &&
-        currentURL.startsWith('/settings/device-settings/')) {
+    if (
+      Object.keys(devices).length === 0
+      && (
+        currentURL.startsWith('/settings/device-settings/')
+        || currentURL.startsWith('/manage-backups/')
+      )
+    ) {
       navigate('/');
```

Comment:
i don't think "wipe history" is a thing? or is that possible? if so that sounds like a good idea. otherwise the user would be stuck in a loop pressing "back" on android, and could not exit the app by pressing back multiple times.

if you `replace: true` here, you will end up in some random location when going back - either back to `/` if you  (e.g. device settings -> manage backups -> unplug -> back), otherwise to wherever you were before you went to device-settings :thinking:

Reply:
below in 

```
 if (isIndex && accounts.length) {
      navigate('/account-summary');
      return;
    }
```

it should be with replace, otherwise the back button will go back to index, which will redirect back to account-summary, etc.?

Reply:
That sounds like it could work!

Reply:
I couldn't get navigate(-100) nor setting `location.href` to work. Could be that clearing history can be done on the Java side, but I am not sure if we want to fix that.

I'll merge this as is.